UpsertedProfileOrchestrator/handler.ts

```diff
@@ -142,18 +129,8 @@ export const getUpsertedProfileOrchestratorHandler = (params: {
           }
         );
 
-        const emailValidationProcessOrchestratorName = pipe(
-          fiscalCode,
-          getIsUserEligibleForNewFeature(
-            cf => params.templateEmailConfig.BETA_USERS.includes(cf),
-            () => false, // NO canary implemented yet
-            params.templateEmailConfig.FF_TEMPLATE_EMAIL
-          ),
-          B.fold(
-            () => "EmailValidationProcessOrchestrator",
-            () => "EmailValidationWithTemplateProcessOrchestrator"
-          )
-        );
+        const emailValidationProcessOrchestratorName =
+          "EmailValidationWithTemplateProcessOrchestrator";
```

Comment:
nit: we can remove this variable since it's only used in a single place.

Reply:
Thanks, fixed in 1f97b73